hawkular-inventory-dist/pom.xml

```diff
@@ -157,6 +169,19 @@
       <!--<version>${version.org.umlg}</version>-->
     <!--</dependency>-->
 
+    <!-- include the versions of the drivers we want -->
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${version.org.postgresql}</version>
```

Comment:
There are 2 options here - either have the driver in WEB-INF/lib (and make sure at build time it is the same version as the module we deploy) - that's how it is done now, or we could depend on the driver module and modify the jboss-deployment-structure to reflect that. The second option is more complicated to do in -dist, because we'd need some xml transformation to do add that into the deployment-structure that the rest.war declares. Not impossible, but this was just simpler (at the cost of slightly bigger distribution size).